### PR TITLE
[RHACS] Updated the contactImageScanners description

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/OSDOCS-3165

Applies to:
3.65, 3.66, 3.67

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@porridge I've created this WIP PR to fix OSDOCS-3165. Can you please help me with the descriptions ?

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 1]
```diff
@@ -63,7 +63,10 @@ The default value is `false`.
 | Use this parameter to bypass the admission controller.
 
 | `admissionControl.contactImageScanners`
-| Specify `true` to enable inline scanning of images that are not already scanned during a deployment's admission review.
+a| Use one of the following values to specify when should the admission controller connect to the image scanner:
```
Comment (porridge) [inline modules/secured-cluster-configuration-options-operator.adoc:66]:
```suggestion
| Use one of the following values to specify when should the admission controller connect to the image scanner:
```

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 2]
```diff
@@ -63,7 +63,10 @@ The default value is `false`.
...
+
+    * `ScanIfMissing` if the image is missing.
```
Comment (porridge) [inline modules/secured-cluster-configuration-options-operator.adoc:68]:
```suggestion
    * `ScanIfMissing` if the scan results for the image are missing.
```

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 3]
```diff
@@ -63,7 +63,10 @@ The default value is `false`.
...
+    * `DoNotScanInline` to not scan the image inline.
```
Comment (porridge) [inline modules/secured-cluster-configuration-options-operator.adoc:69]:
```suggestion
    * `DoNotScanInline` to not scan the image when processing an admission request.
```

**Review by porridge — COMMENTED** [review]
@misberner did I get this right?

**Review by misberner — APPROVED** [review]
LGTM

File: modules/secured-cluster-configuration-options-operator.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/secured-cluster-configuration-options-operator.adoc:66]: Thanks @porridge In AsciiDoc, the `a` before the cell is to allow bullet list inside the table cell.

Comment (porridge) [reply modules/secured-cluster-configuration-options-operator.adoc]: TIL 🤦🏻

**Review by porridge — APPROVED** [review]

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Pull request merged**